cpp/pipelines/add_authority_wikidata_ids.cc

```diff
@@ -51,19 +51,21 @@ void ParseDataDnbFile(std::string input_filename, std::string output_filename) {
         std::string line;
         std::string act_gnd;
         std::string act_name;
-        std::string act_wiki;
+        std::string act_wikidata;
+        std::string act_wikipedia;
         bool sameAs_reached(false);
         bool read_preferred_name(false);
         bool read_gnd_id(false);
         while (std::getline(input_file, line)) {
             if (line == "}, {") {
-                if (not act_wiki.empty() and not act_name.empty() and not act_wiki.empty()) {
-                    output_file << "Name: " << act_name << " GND: " << act_gnd << " Wikidata: " << act_wiki << "\n";
-                    std::cout << "Name: " << act_name << " GND: " << act_gnd << " Wikidata: " << act_wiki << "\n";
+                if (not act_wikidata.empty() and not act_name.empty() and not act_wikidata.empty()) {
```

Comment:
duplicate condition `not act_wikidata.empty()`. Is the third condition meant to be `not act_wikipedia.empty()`?

Reply:
Thanx, yes this should be act_wikipedia.empty()